only run system_time_conversions() test on 64bit systems.
it fails on 32bit systems with

---- time_store::test::system_time_conversions stdout ----
thread 'time_store::test::system_time_conversions' panicked at crates/tor-hsservice/src/time_store.rs:524:38:
2038-01-19T03:14:08Z: OutOfRange

diff --git a/crates/tor-hsservice/src/time_store.rs b/crates/tor-hsservice/src/time_store.rs
index d63d6f015..fafe806d0 100644
--- a/crates/tor-hsservice/src/time_store.rs
+++ b/crates/tor-hsservice/src/time_store.rs
@@ -515,6 +515,7 @@ mod test {
         assert_eq!(e, FutureTimestamp::from_str("T+23kg"));
     }
 
+    #[cfg(target_pointer_width = "64")]
     #[test]
     #[allow(clippy::unusual_byte_groupings)] // we want them to line up, dammit!
     fn system_time_conversions() {
